### ISSUE TITLE
Wrong temperatures and the slider banished

### ISSUE THREAD

**Issue by rafal52** [hirschmann/nbfc#127]
thats happened a few days ago i dont knw how to solve itt and my pc now is a furnace :P  temperature says:
Temperature (cpu): -2147483648°C

My laptop
Win 10 64 bits
asus x550jx i7 4750hq 2.0ghz  8 gb  ram
GTX 950m

**Comment by hirschmann** [comment]
Has NBFC ever worked properly on your notebook?
Have you made any changes to your system (e.g. installed hardware monitoring tools) which  which could interfere with NBFC?

**Comment by rafal52** [comment]
nbfc used to work properly, but i noticed my problem started after the last windows 10 update, i rolled back (returned to my previous windows version) to the last windows version and now the progam works nice.

And i want to share my configuration (for asus x550jx) 
Fan configuration 
Read register 153 | Write register 153 | min spd 15 | max spd 15

Register Write configuration
Write occasion: Onwritefanspeed | Write mode Set | Register 166 | Value 99
That make my fan reach the maximum speed constantly



Thank you for replying my issue and sorry about my english i'm from Uruguay ;)

**Comment by nybbles2** [comment]
Hi,

I just had this happen to me to after udpating to version 1.6.2 beta.

I tried reverting back to 1.6.1 beta, still had the problem..

After trying every possible thing possible... I've gotten the 1.6.1 beta to work again after doing the following:

Install HP Notifications (came through HP Support Assistant Updater)
Uninstall HP Notifications
Reboot
Now it works again...
Reinstall HP Notifications... still works after rebooting
Try to update to 1.6.2... no longer working.
I tried the above with 1.6.2 and it won't work, but when I try the above with 1.6.1 it works again...

I really don't get it... Very confusing.
